chord_metadata_service/mohpackets/apis/core.py

```diff
@@ -235,8 +231,16 @@ def authenticate(self, request, service_token):
 settings_module = os.environ.get("DJANGO_SETTINGS_MODULE")
 # Use OPA in prod/dev environment
 if "dev" in settings_module or "prod" in settings_module:
+    from candigv2_logging.logging import CanDIGLogger, initialize
```

Comment:
What would happen if you just used the CanDIGLogger class for local?

Reply:
I believe it still works, but having the default logging is more convenient for me since it integrates well with the VSCode extension and auto-suggestions.